examples/autotest-validate/Makefile

```diff
@@ -2,10 +2,14 @@ SRC := autotest-validate.c autotest-validate-main.c
 TARGET = autotest-validate
 OBJS := $(SRC:.c=.o)
 
+
 all: $(TARGET)
 
 $(TARGET) : $(OBJS)
 	$(CC) $(CFLAGS) $(INCLUDES) $(OBJS) -o $(TARGET) $(LDFLAGS)
 
 clean:
 	-rm -f *.o $(TARGET) *.elf *.map
+
+
+
```

Comment:
@Sona-Subair look at your git diffs on commit and avoid checking in changes like this which just add whitespace.